content/post/proxysql-one-2022.06.01/index.md

```diff
@@ -0,0 +1,536 @@
+* [Create a test user, test database and table in MySQL server](#create-test-elements)
+* [Put an entry for the test user in ProxySQL server](#user-entry)
+* [Send load over the ProxySQL cluster as the test user](#send-load)
+* [Check the load distribution over the ProxySQL cluster](#check-load-distribution)
+* [Check TLS on the ProxySQL backend and frontend connection](#check-tls)
+
+
+#### <a id="create-test-elements"></a>Create test user, test database and table in MySLQ server
+* Exec into the MySQL primary pod with the following command, we will end up with something like this,
+```shell
+~ $ kubectl exec -it -n demo mysql-server-0 -- bash                            
+Defaulted container "mysql" out of: mysql, mysql-coordinator, mysql-init (init)
+root@mysql-server-0:/#
+```
+* Log into the MySQL console with root password.
+```shell
+root@mysql-server-0:/# mysql -uroot -p$MYSQL_ROOT_PASSWORD
+mysql: [Warning] Using a password on the command line interface can be insecure.
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 639
+Server version: 5.7.36-log MySQL Community Server (GPL)
+
+Copyright (c) 2000, 2021, Oracle and/or its affiliates.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+mysql>
+```
+* Let's get the current users.
+```shell
+mysql> select user from mysql.user;
++---------------+
+| user          |
++---------------+
+| proxysql      |
+| repl          |
+| root          |
+| mysql.session |
+| mysql.sys     |
+| root          |
++---------------+
+7 rows in set (0.00 sec)
+```
+note : the use `proxysql` is the monitor user for ProxySQL created by KubeDB operator
+* Now create the test user, the database and the table and grant privileges to the test user.
+```shell
+mysql> create user 'test'@'%' identified by 'test' REQUIRE SSL;
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> select user from mysql.user;
++---------------+
+| user          |
++---------------+
+| proxysql      |
+| repl          |
+| root          |
+| test          |
+| mysql.session |
+| mysql.sys     |
+| root          |
++---------------+
+7 rows in set (0.00 sec)
+
+mysql> create database random;
+Query OK, 1 row affected (0.00 sec)
+
+mysql> use random;
+Database changed
+
+mysql> create table randomtb(color varchar(120), price integer, primary key(color));
+Query OK, 0 rows affected (0.02 sec)
+
+mysql> grant all privileges on random.* to 'test'@'%';
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> flush privileges;
+Query OK, 0 rows affected (0.01 sec)
+```
+
+#### <a id="user-entry"></a> Put entry for test user in ProxySQL
+
+* Exec into any ProxySQL pod with the following command, we will end up with something like this,
```

Comment:
Exec into any ProxySQL pod and Login to the ProxySQL admin console

Reply:
merged